app/src/forms/rbac/service.js

```diff
@@ -221,6 +234,34 @@ const service = {
     }
   },
 
+  modifySubmissionUser: async (formSubmissionId, userId, body, currentUser) => {
+    if (!userId || !body.permissions) {
+      throw new Problem(422, 'User ID or permissions missing from request');
+    }
+
+    let trx;
+    try {
+      trx = await transaction.start(FormSubmissionUser.knex());
+      // remove existing mappings for the user...
+      await FormSubmissionUser.query(trx)
+        .delete()
+        .where('formSubmissionId', formSubmissionId)
+        .where('userId', userId);
+
+      // create the batch and insert. So if permissions is empty it removes the user from the submission
+      if (body.permissions !== []) {
+        // add ids and save them
+        const items = body.permissions.map(perm => { return { id: uuidv4(), formSubmissionId: formSubmissionId, userId: userId, createdBy: currentUser.username, permission: perm }; });
```

Comment:
This line is a bit long. May be worth indenting:
```js
const items = body.permissions.map(perm => ({
  id: uuidv4(),
  formSubmissionId: formSubmissionId,
  userId: userId,
  createdBy: currentUser.username,
  permission: perm
}));
```

Reply:
Fixed